Makefile

```diff
@@ -3,7 +3,7 @@
 REPO_NAME = go-plugins-helpers
 REPO_OWNER = docker
 PKG_NAME = github.com/${REPO_OWNER}/${REPO_NAME}
-IMAGE = golang:1.7
+IMAGE = golang:1.13
```

Comment:
Should we not use 1.15 ?

Reply:
Yep this is what we do. I think you forgot to refresh your page.